src/irrlicht_changes/CGUITTFont.cpp

```diff
@@ -30,9 +30,15 @@
    john@suckerfreegames.com
 */
 
-#include <irrlicht.h>
 #include <iostream>
 #include "CGUITTFont.h"
+#include "CMeshBuffer.h"
+#include "IFileSystem.h"
+#include "IGUIEnvironment.h"
+#include "IMeshManipulator.h"
+#include "ISceneNode.h"
+#include "IMeshSceneNode.h"
+#include "ISceneManager.h"
```

Comment:
Optional goal: Alphabetical order to find things faster.

Reply:
Won't do it everywhere though, to not blow up changed lines.